### PR TITLE
[PLAT-13018] CI Full Run

### PR DESCRIPTION
**Pull request by joshedney**
## Goal

Add a full CI  pipeline to the existing pipeline setup. 

## Testing

Covered by CI

### CODE REVIEW

File: .buildkite/pipeline_trigger.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+
+if [[ "$BUILDKITE_MESSAGE" == *"[full ci]"* ||
+  "$BUILDKITE_PULL_REQUEST_BASE_BRANCH" == "master" ||
```
Comment (twometresteve) [inline .buildkite/pipeline_trigger.sh:4]:
```suggestion
  "$BUILDKITE_PULL_REQUEST_BASE_BRANCH" == "main" ||
```

Comment (joshedney) [reply .buildkite/pipeline_trigger.sh:4]: 🤦🏼 forgot

File: .buildkite/pipeline.full.yml [hunk 2]
```diff
@@ -0,0 +1,604 @@
...
+          docker-compose#v4.8.0:
+            pull: maze-runner-bs
+            run: maze-runner-bs
+            service-ports: true
+            command:
+              - "--app=@build/bs_xcframework_ipa_url.txt"
+              - "--device=IOS_17"
+              - "--fail-fast"
+              - "--farm=bs"
+              - "--exclude=features/default/automatic_spans_generic_view_load.feature"
```
Comment (twometresteve) [inline .buildkite/pipeline.full.yml:236]: `--exclude` isn't needed as we are only running a single feature anyway.  Same in a few other places too.

**Review by twometresteve — COMMENTED** [review]
Looks good in general, just a few small things.

**Review by twometresteve — APPROVED** [review]

**Review by twometresteve — APPROVED** [review]

**Review by kstenerud — APPROVED** [review]

**Pull request merged**